src/modules/app-db/repositories/admin.repository.ts

```diff
@@ -460,6 +460,17 @@ export class AdminRepository {
     });
   }
 
+  async getNftDetailTrial(id: NftId) {

```

Comment:
There are already functions that return details:
getNftDetail, getCollectionDetail, getWalletDetail

Reply:
@mdevecka These functions are bound to UserID. I need it not to be bound for trial mint. Because these artworks are minted into different wallet that is not owned by the user.

Reply:
If I understand this correctly trial is always located in eva wallet, so we could filter by EvaGalleryWalletAddress instead of user then

Reply:
Will modify, so Eva gallery wallet address and nft id.